test/bootstrap.php

```diff
@@ -5,8 +5,8 @@
 
 // Set API key if none is defined in the env
 // We want the env value to take precedence to enable testing against QA
-if ( !isset($_ENV['TRAACKR_API_KEY']) ) {
+if ( getenv('TRAACKR_API_KEY') === FALSE ) {
    Traackr\TraackrApi::setApiKey('5adab9df789c2147116881f36785f6c3');
```

Comment:
@dchancogne shouldn't we still remove this?

Reply:
This allows the unit tests to work out-of-the-box w/o doing anything special.